protocol/src/main/java/net/md_5/bungee/protocol/packet/BossBar.java

```diff
@@ -18,7 +22,7 @@ public class BossBar extends DefinedPacket
 
     private UUID uuid;
     private int action;
-    private BaseComponent title;
+    private Deserializable<Either<String, SpecificTag>, BaseComponent> title;
```

Comment:
I think every instance is "Deserializable<Either<String, SpecificTag>, BaseComponent>", so we should probably create a helper "ChatDeserializable extends Deserializable<Either<String, SpecificTag>, BaseComponent>"

Reply:
Creating a ChatDeserializable is unfortunaly not working as an alias and would require all implementations to be doubled.
If you want it shorter, the only solution I see is to get rid of generics.

Reply:
It's a tough one. Maybe leave Deserializable and specialise the implementations?